rhel/SPECS/percona-qan-app.spec

```diff
@@ -46,43 +44,5 @@ cp -pav ./dist/qan-app/*    %{buildroot}%{_datadir}/%{name}
 
 
 %changelog
-* Mon Jun 25 2018 Mykola Marzhan <mykola.marzhan@percona.com> - 1.12.0-3
```

Comment:
I suggest not removing changelog as it was detailed

Reply:
It is also wasn't updated for more than a year, and no-one cared. A detailed history is available in git. It is also a PMM Server spec. 😛  All in all, we will remove it.